src/Panel.tsx

```diff
@@ -4,7 +4,11 @@ import { AddonPanel } from "@storybook/components";
 import { ADDON_ID, EVENTS } from "./constants";
 import { PanelContent } from "./components/PanelContent";
 
-export const Panel = (props) => {
+type PanelProps = {
+  active: boolean;
+};
+
+export const Panel = (props: PanelProps) => {
```

Comment:
```suggestion
export const Panel: FC<PanelProps> = (props) => {
```

And suggest the same for the others